EnglishFilter/sections/foreign.txt

```diff
@@ -2176,6 +2176,8 @@ meownime.id###floating_banner_top
 meownime.id##a[href][target="_blank"][rel="nofollow"] > img
 meownime.id##a[href][target="_blank"][rel="noopener"] > img
 ||blogspot.com/*/*/*.gif$domain=meownime.id
+! https://github.com/AdguardTeam/AdguardFilters/issues/28670
+oploverz.in##.separator > a > img
```

Comment:
I think it would be better to add `oploverz.in###btm_banner`, because `.separator > a > img` doesn't hide invisible overlay at the bottom.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/55150478-7beced80-514c-11e9-9b61-935e52dd4b48.png)

</details>